src/Hedgehog/Property.fs

```diff
@@ -81,9 +81,10 @@ module Property =
     let bind (k : 'a -> Property<'b>) (m : Property<'a>) : Property<'b> =
         bindGen (toGen << k) (toGen m) |> ofGen
 
+    let handle (e : exn) =
```

Comment:
Let's make this private or internal until we get a better name. As an alternative, let's get a better name now 😅

Reply:
Made it private.  It probably doesn't ever need to be public.